lib/ACL/RuleManager.php

```diff
@@ -293,6 +295,20 @@ public function saveRule(Rule $rule): void {
 				->andWhere($query->expr()->eq('mapping_type', $query->createNamedParameter($rule->getUserMapping()->getType())))
 				->andWhere($query->expr()->eq('mapping_id', $query->createNamedParameter($rule->getUserMapping()->getId())));
 			$query->executeStatement();
+
+			if ($rule->getUserMapping()->getType() === 'user') {
+				$logMessage = 'The ACL rule was updated for file/folder with id: "%s" for the user "%s" with permission "%s" and mask "%s"';
```

Comment:
Maybe change to this so we group the "to" and "for" together
"The ACL rule was updated to permission "%s" and mask "%s" for file/folder with id %s for user "%s""